octodns_constellix/__init__.py

```diff
@@ -746,30 +847,61 @@ def populate(self, zone, target=False, lenient=False):
         )
         return exists
 
-    def _is_healthcheck_configured(self, record):
-        sonar_healthcheck = record._octodns.get('constellix', {}).get(
-            'healthcheck', None
-        )
-        return sonar_healthcheck is not None
+    def _seconds_to_sonar_interval(self, seconds):
+        # If in doubt, prefer the next larger (cheaper) interval
```

Comment:
@ross @viranch to be discussed if this is reasonable.

Reply:
So long as it's documented it seems reasonable.

Reply:
It is documented in the `README.md`.